storage/volumes.md

```diff
@@ -81,6 +81,20 @@ If you need to specify volume driver options, you must use `--mount`.
   - The `volume-opt` option, which can be specified more than once, takes a
     key-value pair consisting of the option name and its value.
 
+> **Important:** If your volume driver accepts a comma-separated list as an option,
```

Comment:
not sure about indentation here.  let's take a look at the CI build before merging.